src/analysis/sampling/CHRR/chrrSampler.m

```diff
@@ -1,45 +1,40 @@
-% Generate uniform random flux samples with CHRR
-% Coordinate Hit-and-Run with Rounding
+function [samples,roundedPolytope] = chrrSampler(model,numSkip,numSamples,toRound,roundedPolytope,minFlux,maxFlux)
+% CHRRSAMPLER Generate uniform random flux samples with CHRR
+%   Coordinate Hit-and-Run with Rounding
 %
-% USAGE:
+% [samples,roundedPolytope,minFlux,maxFlux] = chrrSampler(model,numSkip,numSamples,toRound,roundedPolytope,minFlux,maxFlux);
 %
-%    [samples, roundedPolytope, minFlux, maxFlux] = chrrSampler(model, numSkip, numSamples, toRound, roundedPolytope, minFlux, maxFlux, useFastFVA);
+%   chrrSampler will generate numSamples samples from model, taking
+%   numSkip steps of a random walk between each sample
 %
-% INPUTS:
-%    model:               COBRA model structure with fields:
+%   Rounding the polytope is a potentially expensive step. If you generate multiple rounds
+%   of samples from a single model, you can save roundedPolytope from the first round and
+%   input it for subsequent rounds.
 %
-%                           * .S - The `m x n` stoichiometric matrix
-%                           * .lb - `n x 1` lower bounds on fluxes
-%                           * .ub - `n x 1` upper bounds on fluxes
-%                           * .c - `n x 1` linear objective
-%    numSkip:             Number of steps of coordinate hit-and-run between samples
-%    numSamples:          Number of samples
+% INPUTS:
+% model ... COBRA model structure with fields:
+% .S ... The m x n stoichiometric matrix
+% .lb ... n x 1 lower bounds on fluxes
+% .ub ... n x 1 upper bounds on fluxes
+% .c ... n x 1 linear objective
+% numSkip ... Number of steps of coordinate hit-and-run between samples
+% numSamples ... Number of samples
 %
 % OPTIONAL INPUTS:
-%    toRound:             {0, 1} Option to round the polytope before sampling.
-%    roundedPolytope:     The rounded polytope from a previous round of
-%                         sampling the same model.
-%    minFlux, maxFlux:    `n x 1` flux minima and maxima from flux variability
-%                         analysis of the same model.
+% toRound ... {0,(1)} Option to round the polytope before sampling.
+% roundedPolytope ... The rounded polytope from a previous round of
+%                     sampling the same model.
+% minFlux,maxFlux ... n x 1 flux minima and maxima from flux variability
+%                     analysis of the same model.
 %
 % OUTPUTS:
-%    samples:             `n x numSamples` matrix of random flux samples
-%    roundedPolytope:     The rounded polytope. Save for use in subsequent
-%                         rounds of sampling.
-%    minFlux, maxFlux:    flux minima and maxima
-%
-% .. Authors: - Ben Cousins and Hulda S. Haraldsdóttir, March 2017
+% samples ... n x numSamples matrix of random flux samples
+% roundedPolytope ... The rounded polytope. Save for use in subsequent
+%                     rounds of sampling.

```

Comment:
Please keep to the defined Documentation scheme detailed in [this](https://github.com/opencobra/cobratoolbox/blob/master/docs/source/guides/documentationGuide.md) guide